core/ts/Poll.ts

```diff
@@ -31,7 +30,6 @@ const blankStateLeafHash = blankStateLeaf.hash();
 interface IPoll {
   topupMessage(_message: Message): void;
   publishMessage(_message: Message, _encPubKey: PubKey): void;
-  mergeAllMessages(): void;
   hasUnprocessedMessages(): boolean;
   processMessages(_pollId: number): unknown;
```

Comment:
hadn't noticed this before, but it should not be unknown, it can cause issues on other packages that depend on it

Reply:
also it looks that some functions are set to return void like tallyVotes - for now maybe just add any before we create better typings?

example of what went wrong on another PR due to this:

```
 ts/__tests__/TallyVotes.test.ts(119,63): error TS2339: Property 'length' does not exist on type 'unknown'.
```

That is using the return value of tallyVotes (the circuit inputs)

Reply:
Sure thing